1-Datasets/reanalyses/ECMWF/1hr_ecmwf_era5-single-levels_GLOBAL.ncml

```diff
@@ -16,13 +16,13 @@
         <netcdf>
 
             <aggregation dimName="time" type="joinExisting" recheckEvery="1 day">
-                <scan location="/pavics-data/ecmwf/era5" regExp="tas.*\.nc$" subdirs="false"/>
+                <scan location="/pavics-data/ECMWF/ERA5/era5-single-levels/GLOBAL/1hr/tas" regExp="tas.*\.nc$" subdirs="false"/>
```

Comment:
`/data/datasets/ECMWF/ERA5` (Thredds might be case-sensitive) do not exist on Boreas.  I assume it exists on Notos?

If not too much trouble, could you ensure any layout on disk refactoring you did on Notos, that Boreas has the same?

The goal of this PR is that it is compatible with both Boreas and Notos at the same time so we can actually merge this PR before Notos go-live, decreasing the "big-bang" effect of the go-live.

"disk2" has been added to Boreas so this PR can also work on Boreas https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/catalog/birdhouse/disk2/catalog.html.

I am aiming for a gradual transition instead of a breaking transition.  This also decrease the need for "synchronized merge" which makes the go-live sensible to timing of the merges.

Reply:
@tlogan2000 tu peux vérifier si ce diff peut être éviter entre Boreas et Notos?  J'aimerais merger ce PR et le déployer sur Boreas.  Si tout reste pareil, sauf les extra `/disk2` path, ce sera 100% compatible avec Boreas.

Reply:
@tlvu I think I will just go back to the previous structure for now  (both location on disk and with ncml file name)..  I can adjust once I start adding new data to the ECMWF folder

Reply:
I will ping you once the change is made

Reply:
@tlvu  I think we should be ok now